src/component/Nav.js

```diff
@@ -0,0 +1,23 @@
+import './styles/Nav.css';
+import React from 'react';
+import planetImg from '../assets/planet.png';
+
+const Nav = () => (
+  <nav>
+    <div className="nav-header">
+      <div className="logo-container">
+        <img
+          src={planetImg}
+          alt="planet logo"
+          className="logo"
+          width="50"
+          height="50"
+        />
+        <h2>Space Travelers Hub</h2>
+      </div>
+    </div>
+    <hr />
+  </nav>
```

Comment:
@Sammy3000 You have done an amazing job in creating this nav component 👏🙂. However, you should use appropriate semantic tags in line with the project description. This is a header section, so instead of using a `nav` tag, it's better to use a `header` tag. 

One of the major reasons for using the right semantic tags is that it promotes web accessibility.

You can read more on Semantic tags [here](https://www.simplilearn.com/tutorials/html-tutorial/html-semantics)